ui/src/components/data-entities/GenericDetailsView.js

```diff
@@ -30,6 +30,13 @@ const renderError = (msgArray) => {
   return (msgArray.length > 0) ? <><Box><Alert severity="error" variant="filled">{msgArray}</Alert></Box></> : <></>;
 };
 
+const nonGenericEntities = {
+  'stagedJobs': {
+    linkLabel: 'Edit Job  ',
+    linkPath: 'PATHTOBECONFIRMED/stagedJobs/{}'
+  }
+};
+
```

Comment:
can you detail why stagedJob isn't a generic entity?

link label 'edit' should be enough

Reply:
This demonstrates it will fit the edit endpoints. I doubt that will be a generic edit form

Reply:
There is no requirement to Edit the jobs.  They are created by the upload/stage process & then only the STATUS field is updated by the data ingest backend process.